reference/src/glossary.md

```diff
@@ -119,6 +136,12 @@ Note that layout does not capture everything that there is to say about how a ty
 Note: Originally, *layout* and *representation* were treated as synonyms, and Rust language features like the `#[repr]` attribute reflect this. 
 In this document, *layout* and [*representation*][representation relation] are not synonyms.
 
+### Memory Address
+[memory address]: #memory-address
+
+A *memory address* is an integer value that identifies where in the physical machine memory some data is stored.
```

Comment:
Bare-metal programmers may confuse this with virtual/physical memory addresses from paging.

Is the term physical machine already established in this context?

Reply:
Ah right, I meant "physical" in contrast to the Abstract Machine, but there's always more layers of abstractions further down the stack. I adjusted the wording, is it better now?

Reply:
Great, thank you! :)